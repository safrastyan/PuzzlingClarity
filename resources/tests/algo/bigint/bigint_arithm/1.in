5
5550555 / 555
5550001 / 555
515 / 5
5550554 / 555
111 / 5