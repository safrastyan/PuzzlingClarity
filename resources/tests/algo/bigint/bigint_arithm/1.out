10001
10000
103
10000
22